packages/nimble-components/src/select/index.ts

```diff
@@ -476,11 +481,14 @@ export class Select
                 o => !o.disabled
             );
             if (enabledOptions.length > 0) {
-                enabledOptions[0]!.selected = true;
+                const selectedOption = enabledOptions.find(
+                    o => o === this.committedSelectedOption
+                ) ?? enabledOptions[0]!;
+                selectedOption.ariaSelected = 'true';
```

Comment:
I think it would be nice if you could scroll to the item that is being marked as active. How difficult would that be?

Reply:
Done.

Reply:
Not difficult, but testing it with a unit test seems particularly challenging given the fact that the option will be scrolled into view in an rAF call. I verified manually.